pkg/gator/verify/runner.go

```diff
@@ -327,7 +364,32 @@ func (r *Runner) runReview(ctx context.Context, newClient func() (gator.Client,
 		Object: *toReview,
 		Source: mutationtypes.SourceTypeOriginal,
 	}
-	return c.Review(ctx, au, reviews.EnforcementPoint(util.GatorEnforcementPoint))
+
+	review, err := c.Review(ctx, au, reviews.EnforcementPoint(util.GatorEnforcementPoint))
```

Comment:
As a general rule in golang you should return ASAP on error.

Reply:
do we always review the au before expansion?

Reply:
I believe we do for at least gator test and the admission webhook.